cmd/burrow/commands/config_options.go

```diff
@@ -10,102 +10,129 @@ import (
 )
 
 type configOptions struct {
-	validatorIndexOpt      *int
-	validatorAddressOpt    *string
-	validatorPassphraseOpt *string
-	validatorMonikerOpt    *string
-	externalAddressOpt     *string
+	configFileOpt        *string
+	genesisFileOpt       *string
+	accountIndexOpt      *int
+	accountAddressOpt    *string
```

Comment:
I'm not sure about the term 'account' for the validator address... we have many accounts, what's special about this one?

Reply:
How about `node`? I would just like to signify, particularly with bond/unbond, that the burrow process may not be running as a validator. However this is currently only true of accounts known in genesis.

Reply:
I've separated these review comments into #1113